gateway-js/src/__tests__/gateway/endToEnd.test.ts

```diff
@@ -8,9 +8,10 @@ import { GraphQLSchemaModule } from '@apollo/subgraph/src/schema-helper';
 import { buildSchema, ObjectType, ServiceDefinition } from '@apollo/federation-internals';
 import gql from 'graphql-tag';
 import { printSchema } from 'graphql';
-import LRUCache from 'lru-cache';
+import { InMemoryLRUCache } from '@apollo/utils.keyvaluecache';
 import { QueryPlan } from '@apollo/query-planner';
 import { createHash } from '@apollo/utils.createhash';
+import { QueryPlanCache } from '@apollo/query-planner/src/config'
```

Comment:
We can't have imports from `src` directories (I know this is just an autocomplete / auto import). You can either import `QueryPlannerConfig` and access the type on it `QueryPlannerConfig['cache']` or export `QueryPlanCache` from the top-level index file (where `QueryPlannerConfig` is exported). I'm comfortable with either approach (I think the 2nd option is probably most helpful for users).

Reply:
Thanks @trevor-scheer resolved